dags/bqetl_ctxsvc_derived.py

```diff
@@ -82,6 +82,22 @@
         arguments=["--schema_update_option=ALLOW_FIELD_ADDITION"],
     )
 
+    contextual_services_derived__event_aggregates_check__v1 = bigquery_etl_query(
+        task_id="contextual_services_derived__event_aggregates_check__v1",
+        destination_table=None,
+        dataset_id="contextual_services_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="wstuckey@mozilla.com",
+        email=[
+            "ctroy@mozilla.com",
+            "telemetry-alerts@mozilla.com",
+            "wstuckey@mozilla.com",
+        ],
+        date_partition_parameter="submission_date",
+        depends_on_past=False,
+        sql_file_path="sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_check_v1/query.sql",
+    )
+
     contextual_services_derived__event_aggregates_spons_tiles__v1 = bigquery_etl_query(
```

Comment:
@skahmann3 do you think we should have a check for this table and for `contextual_services_derived__event_aggregates_suggest__v1` as well? or do you think just having a check for event aggregates is sufficient? We're going to start deprecating event aggregates soon (not removing, but strongly encouraging users to not use this table).

Reply:
@skahmann3 thanks for your thoughts!

@scholtzan We'd like to check that the event aggregates table has >0 rows for each partition and each value of `source` (a field in the event aggregates table that can either be `topsites` or `suggest`). Would it be possible to do that?

Reply:
It will be easier to check the `event_aggregates_suggest` and `event_aggregates_tiles` separately since the partition information which we use here doesn't have access to the `source` field values.

I'll update the check to test both of these datasets.